StudentScheduleInvoker.java

```diff
@@ -0,0 +1,52 @@
+package org.txstate.auto_batcher;
+import java.lang.UnsupportedOperationException;
+import java.io.IOException;
+/**
+ * StudentScheduleInvoker.java : Will compaile the Student ID into a java command to run auto_scheduler
+ * @param N/A
+ * @return N/A
+ * @exception N/A
+ * @version 1.0
+ * @since   5-2-2019
+ * */
+public class StudentScheduleInvoker {
+    String autoSchedulerJarPath;
+    StudentScheduleInvoker(String autoSchedulerJarPath) {
+        this.autoSchedulerJarPath = autoSchedulerJarPath;
+    }
+
+    /**
+     * MultiStudentScheduleInvoker : Will enter a custom command for the respective id
+     * @param Collection<Student> students
+     * @return N/A
+     * @exception IOException
+     */
+    public void schedule(String studentId) {
+        // Example command:
+        // java auto_scheduler-1.0.jar --schedule AXYZID
+        String pathexe = "";
+        pathexe = pathMaker(studentId);
+        System.out.println("Made it");
```

Comment:
Please get rid of the extra debug statements.
or if you have time to be fancy you can enable them by the --Verbose argument to autobatcher.